disperser/encoder/server_v2.go

```diff
@@ -31,23 +35,38 @@ type EncoderServerV2 struct {
 	logger      logging.Logger
 	prover      encoding.Prover
 	metrics     *Metrics
+	grpcMetrics *grpcprom.ServerMetrics
 	close       func()
 
 	runningRequests chan struct{}
-	requestPool     chan struct{}
+	requestPool     chan blobRequest
```

Comment:
nit: can we rename "pool" to "queue" (e.g. queueingRequests, and ServerConfig.RequestQueueSize) so it's consistent with the rest parts?

Reply:
Done V2 code will use new request queue flag. Leaving the V1 server code as is though to prevent having to update the config values.